### ISSUE TITLE
Show track statistics

### ISSUE THREAD

**Issue by pstorch** [OpenTracksApp/OSMDashboard#4]
A split the viewport by 80/20 to show some track statistics.
In portrait mode the statistic should show up below the map, in landscape mode the statistics could be shown as a column to the left of the map.

At the moment there is some logic in the main app for unit conversions, display formats and settings (e.g. use metric system, use pace or speed, ...). Should the OSM Dashboard duplicate this logic? Or do we find some other way.

**Comment by dennisguse** [comment]
Good question (and also an ugly one).
About metric/imperial: OpenTracks stores data as metric and also transfers it as metric.
For the time being, I would stick with presenting metric values only (and later copy the code if necessary).
I removed this functionality from OpenTracks some month ago (the implementation heavily pollutes the code base).

About pace/speed: there was already a request to show both and that might actually be the way to go.

So in short: let's keep it simple for the time being?

**Comment by pstorch** [comment]
Ok, let's start with a simple version. In the meantime I'll kick off the f-droid process with a map-only 1.0 version.

**Comment by pstorch** [comment]
I started to add a little bit statistic to the Map and add a share as picture function. @dennisguse, @rgmf what do you think?

![1638221672761](https://user-images.githubusercontent.com/5598124/143946809-836ab6cc-6479-4508-89bc-62ba9ab040ae.png)

It still needs some polishing and I want to add some customization to it without too much code duplication from OT.

**Comment by rgmf** [comment]
I don't (still) read the code but it sounds like a great idea. I love it the feature :love_you_gesture:

**Comment by dennisguse** [comment]
@pstorch Looks awesome!
i would even propose to remove this share image feature from OpenTracks then :D

**Comment by pstorch** [comment]
Some basic statistic is now on the Dashboard.
Things still on my list:
- imperial/metrics
- customisation of which statistic value to show.
- customizable title?

**Comment by dennisguse** [comment]
@pstorch sounds more like nice to haves :)
For the title, the activity type and track name could be shown.

**Comment by pstorch** [comment]
superseeded by #206
